serverlessworkflow/modules/ROOT/pages/cloud/build-and-deploy-with-serverless-operator-on-kubernetes.adoc

```diff
@@ -0,0 +1,191 @@
+----
+
+=== Create a {product_name} Serverless Platform containing the configuration (i.e. registry address, secret) for building your workflows
+
+The {product_name} Serverless Platform CR is the resource used to control the behavior of the {product_name} Serverless Operator.
+It defines the behavior of all Custom Resources (Workflow and Build) in the given namespace.
+
+Since the {product_name} Serverless Operator is installed in global mode, you will need to specify a {product_name} Serverless Platform CR in each namespace where you want the operator to be executed.
+You can find a basic {product_name} Serverless Platform CR example in the `config/samples` folder that you can simply apply to configure your operator.
+
+.Create a {product_name} Serverless Platform CR
+[source,bash,subs="attributes+"]
+----
+kubectl apply -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml -n kogito-workflows
+----
+
+Note: In this Custom Resource, spec.platform.registry.secret is the name of the secret you created just before.
+
+[TIP]
+====
+You can also update "on-the-fly" the {product_name} Serverless Platform CR registry field with this command (change <YOUR_REGISTRY>)
+
+.Create a {product_name} Serverless Platform CR with a specific registry
+[source,bash,subs="attributes+"]
+----
+curl https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml | sed "s|address: .*|address: <YOUR_REGISTRY>"
+----
+
+In order to retrieve the Cluster IP address of Minikube's internal registry use the following command:
+
+.Retrieve Minikube registry internal IP
+[source,bash,subs="attributes+"]
+----
+kubectl get svc registry -n kube-system -ojsonpath='{.spec.clusterIP}'
+----
+====
+
+== Build and deploy your {context} application
+
+You can now send your {product_name} {context} Custom Resource to the operator which includes the {context} definition.
+
+You can find a basic {product_name} {context} Custom Resource in the `config/samples` folder that is defining the link:{kogitogreetinexample_url}[{product_name} {context} Greeting example].
+
+[source,bash,subs="attributes+"]
+----
+kubectl apply -f ttps://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow.yaml -n kogito-workflows
+----
+You can check the logs of the build of your workflow via:
+
+.Get the {product_name} {context} application pod logs
+[source,bash,subs="attributes+"]
+----
+kubectl logs kogito-greeting-builder -n kogito-workflows
+----
+
+The final pushed image must be printed into the logs at the end of the build.
+
+== Check the {product_name} {context} application is running
+In order to check that the {product_name} {context} Greeting application is up and running, you can try to perform a test HTTP call.
```

Comment:
Maybe the user would like to see the result from outside the cluster, with an ingress I guess ... wdyt ?

Reply:
We can also expose the service outside changing switch it to Nodeport with a command like:
`kubectl patch svc greeting -n kogito-workflows -p '{"spec": {"type": "NodePort"}}'`
Do you think that it could be fine since it is only a local test with one service?

Reply:
`NodePort` should do it yes :)

Reply:
Updated accordingly ;-)